Meadowlark-js/services/meadowlark-fastify/src/Service.ts

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { randomUUID } from 'node:crypto';
+import Fastify from 'fastify';
+import type { FastifyInstance } from 'fastify';
+import { deleteIt, get, update, upsert } from './handler/CrudHandler';
+import { metaed, apiVersion, swaggerForResourcesAPI, swaggerForDescriptorsAPI } from './handler/MetadataHandler';
+import { oauthHandler } from './handler/OAuthHandler';
+
+export function buildService(): FastifyInstance {
+  const service = Fastify({
+    logger: true,
+    genReqId: () => randomUUID(),
+  });
+
+  // override json parser to leave body as string
+  service.addContentTypeParser('application/json', { parseAs: 'string' }, (_req, body, done) => {
```

Comment:
Only unusual thing about fastify, by default it parses the body so it can apply JSON Schemas. We override here to pass the body through as a string.